### DISCUSSION TITLE
Tidy Tales - The Pareto Principle in R package development

### DISCUSSION THREAD

**Discussion by giscus** [mccarthy-m-g/tidytales#45] [Comments]
# Tidy Tales - The Pareto Principle in R package development

Exploring R package developer data on CRAN with the {tidyverse} and {gt}.

https://tidytales.ca/posts/2023-05-03_r-developers/

**Comment by EllaKaye** [comment]
This is great! I really enjoyed reading it, learnt a lot from your code, and had fun exploring the table of contributors. Kudos!

**Reply by mccarthy-m-g** [reply]
Thanks Ella! 😄
